lib/web/desktop.go

```diff
@@ -84,6 +121,11 @@ func (h *Handler) handleDesktopAccessWebsocket(
 	serviceConTLS := tls.Client(serviceCon, ctx.clt.Config())
 	log.Debug("Connected to windows_desktop_service")
 
+	// Write username, initial width, and initial height to the windows desktop service using TDP
```

Comment:
I don't think this comment is necessary. It's already pretty clear that we're using the `tdp` package and writing a `ClientUsername` and `ClientScreenSpec`.

I would just remove it, then it won't go out of date if we eventually support other (non-Windows) desktop services.